packages/env/lib/init-config.js

```diff
@@ -235,33 +236,32 @@ RUN apt-get -qy install $PHPIZE_DEPS && touch /usr/local/etc/php/php.ini
 # Set up sudo so they can have root access.
 RUN apt-get -qy install sudo
 RUN echo "$HOST_USERNAME ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers`;
-	} else {
-		dockerFileContent += `
+			if ( shouldEnableXdebug( config ) ) {
+				// Xdebug is only needed for WordPress environments.
+				dockerFileContent += getXdebugSetup( config.xdebug );
+			}
+			break;
+		}
+		case 'cli': {
```

Comment:
We should keep Xdebug support in the CLI container

Reply:
added 👍